custom/icds_reports/templates/icds_reports/icds_app/progress-report.directive.html

```diff
@@ -28,46 +28,21 @@ <h2>No Data</h2>
         </div>
         <div ng-if="!$ctrl.report">
             <div class="row">
-                <div class="col-md-4">
-                    <div class="nav-block pointer col-sm-4 col-md-offset-4 col-md-8" style="background-color: #004ebc;" ng-click="$ctrl.goToReport('maternal_and_child_nutrition')">
-                        <img style="width: 152px;" src="{% static 'img/maternal_and_child.png' %}"/>
-                        <div class="caption">Maternal and Child Nutrition</div>
-                    </div>
-                </div>
-                <div class="col-md-4">
-                    <div class="nav-block pointer col-sm-4 col-md-offset-2 col-md-8 fact-sheet-nav-interventions" ng-click="$ctrl.goToReport('interventions')">
-                        <img src="{% static 'img/interventions.png' %}"/>
```

Comment:
do you happen to have any idea why this image didn't have a fixed width but the other two did?

Reply:
These images are big. So they had it to give it a fixed width to contain it.